guardian/newsletters-nx
# Dblatcher/drafts view

## What does this change?
Adds API routes, route loader and Components to display the Draft newsletters created in the wizard.

## How to test

Create a Draft using the wizard, use the "view draft newsletters" nav link to see it on the list.


## Images

<img width="707" alt="Screenshot 2023-02-24 at 13 01 30" src="https://user-images.githubusercontent.com/30567854/221189627-0901132d-73e1-4570-88c3-ceff2b7f0dc2.png">
<img width="707" alt="Screenshot 2023-02-24 at 13 01 11" src="https://user-images.githubusercontent.com/30567854/221189647-71d3ebcd-733b-426e-8412-681c740e18fc.png">
